extension/src/config/init/ensure-api-extensions.js

```diff
@@ -40,19 +40,13 @@ async function ensureApiExtensions(
         'Successfully created an API extension for payment resource type ' +
           `(key=${apiExtensionTemplate.key})`
       )
-    } else if (
-      !_.isEqual(existingExtension.destination, extensionDraft.destination)
-    ) {
+    } else {
+      const actions = buildUpdateActions(existingExtension, extensionDraft)
```

Comment:
What if actions array is empty? Should not we check it before trying to update?

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/1017/files